src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -73,10 +73,26 @@
     CompletableFuture<InvoiceLine> future = new VertxCompletableFuture<>(ctx);
 
     try {
+      // 1. GET invoice-line from storage
       handleGetRequest(resourceByIdPath(INVOICE_LINES, id, lang), httpClient, ctx, okapiHeaders, logger)
         .thenAccept(jsonInvoiceLine -> {
           logger.info("Successfully retrieved invoice line: " + jsonInvoiceLine.encodePrettily());
-          future.complete(jsonInvoiceLine.mapTo(InvoiceLine.class));
+
+          // 2. Copy invoice-line from storage for future comparison
+          InvoiceLine invoiceLineFromStorage = jsonInvoiceLine.mapTo(InvoiceLine.class);
+          InvoiceLine invoiceLineToRecalculateTotal = jsonInvoiceLine.mapTo(InvoiceLine.class);
+
+          // 3. Calculate invoice-line totals, if different from storage and write it back to storage
+          calculateInvoiceLineTotals(invoiceLineToRecalculateTotal).thenAccept(invoiceLineWithTotalRecalculated -> {
```

Comment:
'calculateInvoiceLineTotals()' is also called in `InvoicesImpl#getInvoiceInvoiceInvoiceLinesById`. Please remove one unnecessary method call.